ci-operator/step-registry/shiftstack/cleanup/shiftstack-cleanup-commands.sh

```diff
@@ -7,4 +7,16 @@ export OS_CLIENT_CONFIG_FILE="${SHARED_DIR}/clouds.yaml"
 export CLUSTER_TYPE="${CLUSTER_TYPE_OVERRIDE:-${CLUSTER_TYPE:-}}"
 SLACK_HOOK="$(</var/run/slack-hooks/shiftstack-bot)"
 
-prune --resource-ttl="$RESOURCE_TTL" --slack-hook="$SLACK_HOOK" --no-dry-run > "${ARTIFACT_DIR}/result.json"
+EXCLUDE_ARG=""
+if [ -n "${IMAGES_TTL:-}" ]; then
+    EXCLUDE_ARG="--exclude=images"
+    prune --resource-ttl="$IMAGES_TTL" --slack-hook="$SLACK_HOOK" --no-dry-run "$EXCLUDE_ARG" --include="images" > "/tmp/images-result.json"
```

Comment:
why should we include and exclude images from the prune?

Reply:
because we would separate the cleanup between images (every 2 weeks) and the rest of the resources (every 7h)

Reply:
In the description of [this PR](https://github.com/shiftstack/prune/pull/26) you mention resource types can not be included and excluded simultaneously. Is that the same that is happening here?

Reply:
> because we would separate the cleanup between images (every 2 weeks) and the rest of the resources (every 7h)

shouldn't it be just --include?

Reply:
in the same command we can't exclude and include at the same time.
so the proposal here is to run 2 prune commands:
* one that only include images
* one that exclude images

that way we can control image TTL and other resources TTL.

Reply:
sorry I fixed my error